spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/tracing/BravePropagationConfigurations.java

```diff
@@ -97,12 +96,10 @@ BaggagePropagation.FactoryBuilder propagationFactoryBuilder(
 			return builder;
 		}
 
-		@SuppressWarnings("deprecation")
 		private Factory createThrowAwayFactory() {
 			return new Factory() {
-
 				@Override
-				public <K> Propagation<K> create(KeyFactory<K> keyFactory) {
```

Comment:
the key is that create hasn't been called by Brave since 5.12 I think. It has been calling `get()` which defaulted to forward to create. `create()` is deprecated and will be removed in Brave 7, so not implementing that here allows less maintenance in the future.